awscli/customizations/s3/subcommands.py

```diff
@@ -290,12 +290,23 @@
                  'Using a lower value may help if an operation times out.')}
 
 
+IGNORE_GLACIER_WARNINGS = {
+    'name': 'ignore-glacier-warnings', 'action': 'store_true',
+    'help_text': (
+        'Turns off glacier warnings. Warnings about operations that cannot '
+        'be performed because it involves copying, downloading, or moving '
+        'a glacier object will no longer be printed to standard error and '
```

Comment:
Changing this to "*and will no longer* cause the return code of the command to be ``2``" might make it a bit more clear that using this option will prevent an error exit rather than cause one.